### PR TITLE
SRVLOGIC-138: [Docs] Update OpenShift Serverless Logic for DP6 release

### PR DESCRIPTION
**Pull request by VaniHaripriya**
Jira : [SRVLOGIC-138](https://issues.redhat.com/browse/SRVLOGIC-138)

Following changes have been added:

-  [KOGITO-8676] OnOverflow management (#315)
-  [KOGITO-8409]Added info about CloudEvents to Knative functions (#312)
-  [KOGITO-8908] Documenting sysout arguments and log level (#321)
-  [KOGITO-8443] Added note about simplified format for Knative service discovery(#326)
-  Updated with steps to overcome an issue on macOS (#274)
-  [KOGITO-8818] Rearrange guides to add operator and quarkus cloud (#313)
-  [KOGITO-8820] Fix references in timeout guide and create a new section for job service(#320)
-  Updated versions in antora.yml file
-  Updated Release notes

### CODE REVIEW

File: modules/serverless-logic/pages/cloud/quarkus/deploying-on-kubernetes.adoc
Comment (radtriste) [inline modules/serverless-logic/pages/cloud/quarkus/deploying-on-kubernetes.adoc:1]: Check the title and the `{context}` string which does not exist:
https://deploy-preview-94--jazzy-shortbread-5f62b7.netlify.app/docs/latest/serverless-logic/cloud/quarkus/deploying-on-kubernetes.html

File: modules/serverless-logic/pages/integrations/custom-functions-knative.adoc [hunk 1]
```diff
@@ -127,10 +127,14 @@ You should see an output like (`id` will change):
 
 --
 
+== Sending as CloudEvent
+
+Knative functions support https://github.com/knative/func/blob/main/docs/function-templates/quarkus.md#invocation-parameters[CloudEvent as the message protocol]. {context} can create and post CloudEvent messages in `functionRef`. For more information see xref:core/custom-functions-support.adoc#sending-cloudevents[]
```
Comment (radtriste) [inline modules/serverless-logic/pages/integrations/custom-functions-knative.adoc:132]:
```suggestion
Knative functions support https://github.com/knative/func/blob/main/docs/function-templates/quarkus.md#invocation-parameters[CloudEvent as the message protocol]. {product_name} can create and post CloudEvent messages in `functionRef`. For more information see xref:core/custom-functions-support.adoc#sending-cloudevents[]
```

File: modules/serverless-logic/pages/persistence/postgresql-flyway-migration.adoc
Comment (radtriste) [inline modules/serverless-logic/pages/persistence/postgresql-flyway-migration.adoc:1]: Check for `{context}` in that file ;)

**Review by radtriste — CHANGES_REQUESTED** [review]
Also `serverless-logic-web-tools-enable-kogito-swf-visualization.adoc` still contain some `{context}` strings

File: modules/serverless-logic/pages/persistence/postgresql-flyway-migration.adoc
Comment (VaniHaripriya) [reply modules/serverless-logic/pages/persistence/postgresql-flyway-migration.adoc:1]: Thank you Tristan, replaced all the {context} to {product_name}.

File: modules/serverless-logic/pages/release-notes.adoc [hunk 2]
```diff
@@ -3,13 +3,13 @@
 
 == Notable changes
 
-* link:https://issues.redhat.com/browse/SRVLOGIC-7[SRVLOGIC-7] - [core] Simplified serverless knative function invocation
-* link:https://issues.redhat.com/browse/SRVLOGIC-47[SRVLOGIC-47] - [use case] Improve Workflow Timeouts support (community only)
 * link:https://issues.redhat.com/browse/SRVLOGIC-62[SRVLOGIC-62] - Data Index as Quarkus extension
```
Comment (MarianMacik) [inline modules/serverless-logic/pages/release-notes.adoc:6]: SRVLOGIC-62 was done as a part of DP5.
```suggestion
```

File: modules/serverless-logic/pages/release-notes.adoc [hunk 3]
```diff
@@ -3,13 +3,13 @@
...
-* link:https://issues.redhat.com/browse/SRVLOGIC-65[SRVLOGIC-65] - Upgrade Workflow Runtime to RHBQ 2.13
-* link:https://issues.redhat.com/browse/SRVLOGIC-66[SRVLOGIC-66] - [SWF Editor] Add support to Async API
-* link:https://issues.redhat.com/browse/SRVLOGIC-67[SRVLOGIC-67] - Serverless Workflow Visualization Improvements - DP5
-* link:https://issues.redhat.com/browse/SRVLOGIC-69[SRVLOGIC-69] - [SWF Dev UI] Add support to start workflows using gateway API and cloud events
-* link:https://issues.redhat.com/browse/SRVLOGIC-76[SRVLOGIC-76] - Data-index prod ephemeral image is missing the `KOGITO_DATA_INDEX_QUARKUS_PROFILE`
+* link:https://issues.redhat.com/browse/SRVLOGIC-136[SRVLOGIC-136] - Recollection of DP6 small features
+* link:https://issues.redhat.com/browse/SRVLOGIC-130[SRVLOGIC-130] - Execute serverless workflow without code generation
+* link:https://issues.redhat.com/browse/SRVLOGIC-129[SRVLOGIC-129] - Jobs Service features, fixes, and enhancements available in the technical preview
+* link:https://issues.redhat.com/browse/SRVLOGIC-132[SRVLOGIC-132] - Jobs Service Embedded Addon
```
Comment (MarianMacik) [inline modules/serverless-logic/pages/release-notes.adoc:10]: SRVLOGIC-129, SRVLOGIC-130, SRVLOGIC-132 were not a part of DP6.
```suggestion
* link:https://issues.redhat.com/browse/SRVLOGIC-68[SRVLOGIC-68] - YAML Serverless Workflow parsing on Stunner
* link:https://issues.redhat.com/browse/SRVLOGIC-133[SRVLOGIC-133] - Support validation of output schema
* link:https://issues.redhat.com/browse/SRVLOGIC-136[SRVLOGIC-136] - Recollection of DP6 small features
* link:https://issues.redhat.com/browse/SRVLOGIC-145[SRVLOGIC-145] - Serverless Workflow Visualization Improvements - DP6
```

File: modules/serverless-logic/pages/release-notes.adoc [hunk 4]
```diff
@@ -3,13 +3,13 @@
...
 
 == Other changes and Bug fixes
+
+* link:https://issues.redhat.com/browse/SRVLOGIC-129[SRVLOGIC-129] - Jobs Service features, fixes, and enhancements available in the technical preview
+* link:https://issues.redhat.com/browse/SRVLOGIC-71[SRVLOGIC-71] - [use case] Add support for Workflow CR devmode in the Serverless Logic Operator
```
Comment (MarianMacik) [inline modules/serverless-logic/pages/release-notes.adoc:15]: SRVLOGIC-71 and SRVLOGIC-129 are not a part of DP6.

File: antora.yml [hunk 5]
```diff
@@ -19,10 +19,10 @@ asciidoc:
...
     vscode_version: 1.66.0
-    kn_cli_version: 0.27.0
+    kn_cli_version: 0.28.0
     docker_min_version: 20.10.7
     docker_compose_min_version: 1.27.2
-    operator_version: v1.35.0
+    operator_version: v1.38.0
     kogito_devservices_imagename: registry.redhat.io/openshift-serverless-1-tech-preview/logic-data-index-ephemeral-rhel8:1.24.0-11
     kogito_examples_repository_url: 'https://github.com/kiegroup/kogito-examples'
     kogito_sw_examples_url: https://github.com/kiegroup/kogito-examples/tree/1.32.x/serverless-workflow-examples
```
Comment (krisv) [inline antora.yml:28]: This probably needs updating to 1.38 as well @VaniHaripriya ?

**Review by krisv — CHANGES_REQUESTED** [review]

File: modules/serverless-logic/pages/release-notes.adoc (hunk 3 above)
Comment (VaniHaripriya) [reply modules/serverless-logic/pages/release-notes.adoc:10]: @MarianMacik Some of the jiras under SRVLOGIC-62,SRVLOGIC-129, SRVLOGIC-130, SRVLOGIC-132  were released in DP6. I got them from the release notes of 1.36,1.37 & 1.38. Please suggest if I need to remove them or keep in  this case.

**Comment by radtriste** [comment]
Is that on purpose that the `Cloud` part is not included into the `Serverless Logic` but rather a totally different menu on the left panel ?
image

File: modules/ROOT/nav.adoc [hunk 6]
```diff
@@ -36,47 +36,57 @@
...
 
 *** Integrations
 **** xref:serverless-logic:integrations/camel-routes-integration.adoc[Integrating with Camel Routes]
-**** xref:serverless-logic:integrations/custom-functions-knative.adoc[Invoking Knative services from {context}]
+**** xref:serverless-logic:integrations/custom-functions-knative.adoc[Invoking Knative services]
 **** xref:serverless-logic:integrations/expose-metrics-to-prometheus.adoc[Exposing the workflow base metrics to Prometheus]
 // **** xref:serverless-logic:integrations/camel-k-integration.adoc[Integrating with Camel-K]
 **** xref:serverless-logic:integrations/serverless-dashboard-with-runtime-data.adoc[Displaying workflow data in dashboards]
+* Supporting Services
+** xref:serverless-logic:supporting-services/jobs-service.adoc[Job Service]
```
Comment (MarianMacik) [inline modules/ROOT/nav.adoc:85]: Same here what Tristan found.
```suggestion
*** Supporting Services
**** xref:serverless-logic:supporting-services/jobs-service.adoc[Job Service]

```

**Comment by VaniHaripriya** [comment]
> Is that on purpose that the `Cloud` part is not included into the `Serverless Logic` but rather a totally different menu on the left panel ? image

Thank you Tristan. Updating the nav doc with correct fromat.

**Review by radtriste — APPROVED** [review]
one minor comment

File: modules/serverless-logic/pages/cloud/index.adoc
[the remark this answers is not in the document]
Comment (VaniHaripriya) [reply modules/serverless-logic/pages/cloud/index.adoc:1]: Fixed this issue.

**Review by ricardozanini — APPROVED** [review]

**Comment by ricardozanini** [comment]
@krisv wanna take a final look?

**Review by MarianMacik — APPROVED** [review]

**Pull request merged**